### DISCUSSION TITLE
Current version is not installing with composer

### DISCUSSION THREAD

**Discussion by fiammybe** [ImpressCMS/impresscms#993] [Q&A] [answered]
Hi, I'm trying to do an installation using composer by using the following command : 
`create-project -s dev impresscms/impresscms `

I get the following errors : 
Problem 1
    - mindplay/composer-locator is locked to version 2.1.3 and an update of this package was not requested.
    - mindplay/composer-locator 2.1.3 requires composer-plugin-api ^1 -> found composer-plugin-api[2.1.0] but it does not match the constraint.
  Problem 2
    - symfony/deprecation-contracts is locked to version v2.4.0 and an update of this package was not requested.
    - You can only install one version of a package, so only one of these can be installed: symfony/deprecation-contracts[v2.4.0].

@MekDrop Any idea where these 2 are? I cannot find them in the composer.json file for the current 2.0 branch.

**Comment by MekDrop** [comment]
I don't use create-project often but I think these problems where fixed in master and is pending release. Maybe some switch is missing? :|

**Reply by fiammybe** [reply]
I did a new install, the issue has changed now visibly : 
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - impresscms/impresscms is present at version 1.0.0+no-version-set and cannot be modified by Composer
    - roave/security-advisories dev-master conflicts with roave/security-advisories dev-master.
    - Root composer.json requires roave/security-advisories dev-master -> satisfiable by roave/security-advisories[dev-master].

**Comment by fiammybe** [comment, marked as answer]
I was able to get it to work by removing the roave/security-advisories from the 'dev' section in the composer.json file. What are we using that for ?

**Reply by MekDrop** [reply]
`roave/security-advisories` is composer plugin for not including any package that has any known security vulnerabilities. I think the issue is somewhere else. 

I did few pull requests and for me it kinda everything installed. I use word 'kinda' because I needed for some reasons to run from console DB migrations command. I hope next week I will find some free time to look into.
